pkg/eval/builtin_special.go

```diff
@@ -345,7 +346,13 @@ func use(fm *Frame, spec string, r diag.Ranger) (*Ns, error) {
 				return nil, err
 			}
 		}
-		path := filepath.Clean(dir + "/" + spec + ".elv")
+		var path string
+		_, plugin := os.Stat(filepath.Clean(dir + "/" + spec + ".so"))
```

Comment:
Name error return value `err`, not something else; this is a strong convention followed in the codebase.